lualib/resty/kong/tls.lua

```diff
@@ -71,8 +71,9 @@ if ngx.config.subsystem == "http" then
         end
 
         local r = get_request()
-        -- no need to check if r is nil as phase check above
-        -- already ensured it
+        if not r then
```

Comment:
It is a repeated pattern. I think we could define a simple function to wrap it, like:
```
local orign_get_request = ngx.get_request
local function get_request()
...
end
```

Reply:
Good idea!